libminifi/src/processors/TailFile.cpp

```diff
@@ -36,6 +36,7 @@
 #include <sstream>
 #include <string>
 #include <iostream>
+#include <utils/file/FileUtils.h>
```

Comment:
We should keep with the current formatting. I would prefer a change too but we should probably avoid making stylistic changes.

Reply:
Good spot, thanks! I was my IDE doing it automatically. Fixed.